yt_dlp/extractor/drtuber.py

```diff
@@ -21,84 +23,201 @@ class DrTuberIE(InfoExtractor):
-            r'poster="([^"]+)"',
-            webpage, 'thumbnail', fatal=False)
+        mobile_webpage = None
+        if not title:
+            mobile_webpage = self._download_webpage(
+                f'http://m.{domain}.com/video/{video_id}',
+                video_id, 'Downloading mobile video page', fatal=False) or ''
+
+            title = strip_or_none(video_data.get('title') or self._html_search_regex(
+                (r'''<span\s[^>]*?\btitle\s*=\s*(?P<q>"|'|\b)(?P<title>[^"]+)(?P=q)\s*>''',
+                    r'''<div\s[^>]*?\bclass\s*=\s*(?P<q>"|'|\b)thumb-holder video(?P=q)>\s*<h5\b[^>]*>(?P<title>[^<]+)</h5''',
+                    r'''<span\s[^>]*?\bclass\s*=\s*(?P<q>"|'|\b)title_thumb(?P=q)>(?P<title>[^<]+)</span'''),
+                mobile_webpage, 'title', group='title'))
+
+        thumbnails = []
+        if not mobile_webpage:
+            mobile_webpage = self._download_webpage(
+                f'http://m.{domain}.com/video/{video_id}',
+                video_id, 'Downloading mobile video page', fatal=False) or ''
+
+            thumbnails = [
+                {'url': thumb_url} for thumb_url in re.findall(
+                    r'<div\s+class\s*=\s*"video-tmb-wrap"\s*>\s*<img\s+src\s*=\s*"([^"]+)"\s*/>', mobile_webpage)
+                if url_or_none(thumb_url)]
+
+        if url_or_none(video_data.get('poster')):
+            thumbnails.append({'url': video_data['poster'], 'preference': 1})
 
         def extract_count(id_, name, default=NO_DEFAULT):
             return str_to_int(self._html_search_regex(
-                rf'<span[^>]+(?:class|id)="{id_}"[^>]*>([\d,\.]+)</span>',
-                webpage, f'{name} count', default=default, fatal=False))
+                rf'<span[^>]+(?:class|id)="{id_}"[^>]*>(?P<{name}>[\d,\.]+)</span>',
+                webpage, f'{name} count', default=default, fatal=False, group=name))
 
-        like_count = extract_count('rate_likes', 'like')
-        dislike_count = extract_count('rate_dislikes', 'dislike', default=None)
-        comment_count = extract_count('comments_count', 'comment')
+        like_count = extract_count('(?:rate_likes|rate_votes|video_rate_votes)', 'like')
+        dislike_count = extract_count('(?:rate_dislikes|rate_votes|video_rate_votes)', 'dislike', default=None)
+        comment_count = extract_count('(?:comments_count|comments__counter)', 'comment')
 
         cats_str = self._search_regex(
-            r'<div[^>]+class="categories_list">(.+?)</div>',
+            r'<div[^>]+class="(?:categories_list|data_categories|video-cat)">(.+?)</div>',
             webpage, 'categories', fatal=False)
-        categories = [] if not cats_str else re.findall(
-            r'<a title="([^"]+)"', cats_str)
+
+        categories = None
+        if cats_str:
+            for pattern in [r'<a[^>]+title="([^"]+)"', r'<a[^>]+href="/categories/([^"]+)"']:
+                categories = re.findall(pattern, cats_str)
+                if categories:
+                    break
```

Comment:
could inline this in the return too
```suggestion
        categories = (re.findall(r'<a[^>]+title="([^"]+)"', cats_str)
                      or re.findall(r'<a[^>]+href="/categories/([^"]+)"', cats_str))
```